elasticdl/README.md

```diff
@@ -44,12 +47,13 @@ docker run --rm -u $(id -u):$(id -g) -it \
 
 ## Continuous Integration Docker Image
 
-Continuous integration docker image contains everything from the development docker image and the ElasticDL source code. It is  used to run continuous integration with the latest version of the source code. In repo's root directory, run the following command:
+Continuous integration docker image contains everything from the development docker image , processed demo data in RecordIO format and the ElasticDL source code. It is  used to run continuous integration with the latest version of the source code. In repo's root directory, run the following command:
```

Comment:
remove unnecessary space before comma and "It is  used"

Reply:
Fixed.